### ISSUE TITLE
Assertion failure not detected in a multi-file include setup

### ISSUE THREAD

**Issue by mschlaipfer** [dafny-lang/language-server-csharp#40]
Using the Dafny plugin v1.3.0 for VSCode, the `assert false` in file F3.dfy below is not highlighted by the IDE. Running Dafny on the command line on file F3.dfy points out the assertion failure.

The setup consists of 3 files: F1.dfy, F2.dfy, and F3.dfy.

F1.dfy:
```
module M1 {}
```

F2.dfy:
```
include "F1.dfy"
```

F3.dfy:
```
include "F1.dfy"
include "F2.dfy"

method m()
{
  assert false;
}
```

When deleting either of the `include` lines in F3.dfy the `assert false` is highlighted. Deleting the `include` from F2.dfy also makes the `assert false` be detected (after changing and saving F3.dfy). Deleting the `module` from F1.dfy also lets the `assert false` be detected.

**Comment by camrein** [comment]
Thank you for your report. I quickly ran your example through the debugger. Your example produces an error that leads to the situation that the verifier is never run:
```
document.Errors.AllMessages[ErrorLevel.Error].Select(message => message.message + " @ " + message.token.filename);
Count = 2
    [0]: "the included file d:\\Data\\Development\\hsr\\dafny-test\\report\\F1.dfy contains error(s) @ d:\\Data\\Development\\hsr\\dafny-test\\report\\F2.dfy"
    [1]: "Duplicate module name: M1 @ d:\\Data\\Development\\hsr\\dafny-test\\report\\F1.dfy"
```
If there are any parser or resolver errors, the verifier is never run; thus, no error about the erroneous assertion is displayed. Side-note: Errors from foreign files are not displayed to prevent unnecessary clutter.

TL;DR There must be some discrepancy between the language server and the Dafny compiler in terms of include resolution that I have to investigate.

**Comment by mschlaipfer** [comment]
Thanks for the quick reply!

**Comment by camrein** [comment]
Fixed in the *master* and in the *dafny-v3.0.0-PreRelease3* branches.
